snewpdag/plugins/Histogram1D.py

```diff
@@ -146,25 +147,34 @@ def summary(self):
             }
 
   def summary_err(self):
-    return {
+    d =    {
             'error_sum': self.error_sum,
             'error_sum2': self.error_sum2,
-            'error_std': np.sqrt( self.error_sum2 / self.count),
             'stats_sum': self.stats_sum,
             'stats_sum2': self.stats_sum2,
-            'stats_std': np.sqrt( self.stats_sum2 / self.count),
             'sys_sum': self.sys_sum,
             'sys_sum2': self.sys_sum2,
-            'sys_std': np.sqrt( self.sys_sum2 / self.count),
             }
+    if self.count == 0:
+      d['error_std'] = math.nan
+      d['stats_std'] = math.nan
+      d['sys_std'] = math.nan
+    else:
+      d['error_std'] = np.sqrt(self.error_sum2 / self.count)
+      d['stats_std'] = np.sqrt( self.stats_sum2 / self.count)
+      d['sys_std'] = np.sqrt( self.sys_sum2 / self.count)
+    return d
```

Comment:
Perhaps better a ZeroDivisionError. We need to catch all the exceptions we can, since ideally snewpdag doesn't crash when the data's partial or a little weird (which seems likely).

Reply:
on second thought, empty histograms do happen in the normal course of events, so I think I'll leave the condition.  "Exceptions should be exceptional"...